docs/01-overview/02-main-areas/service-management/smgt-11-hb-create-addons.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Create addons
```

Comment:
The title implies that this document provides instructions how to create addons, but does it? As a reader, I'd expect a sequence of steps.

Reply:
Changed to `Addons structure`